core/geometry/src/polyface/Polyface.ts

```diff
@@ -140,43 +145,17 @@ export class IndexedPolyface extends Polyface { // more info can be found at geo
       this._facetToFaceData = [];
   }
 
-  /**
-   * Given an array of strictly increasing numbers, find the index of the largest number that is less than or equal
-   * to `value`.
-   * * Get an initial estimate by proportions of `value` and the first and last entries.
-   * * Linear search from there for final value.
-   * * For regularly spaced numbers (e.g., `data` is the `_facetStart` indices for a triangulated mesh or a quad mesh),
-   * the proportional estimate will be immediately correct.
-   */
-  public static searchStrictlyIncreasingNumbers(data: number[], value: number): number | undefined {
```

Comment:
Is it okay to get rid of this public method? Was it added recently or something?

Reply:
I was recent. In the PR description: "this method was initially added in 5.0.0-dev.46, @bentley/imodeljs-native 5.0.40". I am assuming this is recent enough?